backends/github/labels/README.md

```diff
@@ -3,7 +3,7 @@
 | ✅ Auth | ✅ Writes | ❌ Uploads |
 |---------|-----------|-----------|
 
-**URL format** Either `https://github.com/{owner}/{repo}/labels` or any URL that starts with `https://api.github.com/repos/` and ends with `/labels`
+**URL format** Either `https://github.com/{owner}/{repo}/labels` or any URL that starts with either `https://github.com/repos/` or `https://api.github.com/repos/` and ends with `/labels`
```

Comment:
Wait, the github.com URLs don't have `repos`!

Reply:
Yup, they don't. I chose the wrong way to shorten the text. I will fix it ASAP so that it is ready before the public launch.